frontend/src/web.py

```diff
@@ -120,6 +120,12 @@ def main():
     root.hosts = hostsfrontend.HostsFrontend()
     root.api = api.Root(root)   # JSON api exposure, enabling integration with other monitoring tools
 
+    if settings.get('oauth2',{}).get('switch'):
```

Comment:
I thought so too; I discovered yaml-load interprets that to python-true.
Just to stay out of translation issues I decided for exists or not (should cover boolean interpretations too).